### PR TITLE
Add javadocs

### PR DESCRIPTION
**Pull request by blackmirag3**

### CODE REVIEW

File: src/main/java/classify/commands/EditStudent.java [hunk 1]
```diff
@@ -21,6 +21,16 @@ public class EditStudent {
     private static final String DEFAULT_STRING_VALUE = "Unknown";
 
     // @@author blackmirag3
+
+    /***
```
Comment (tayponghee) [inline src/main/java/classify/commands/EditStudent.java:25]: Remove the extraneous *

File: src/main/java/classify/commands/EditStudent.java [hunk 2]
```diff
@@ -58,6 +68,13 @@ public static void editStudent(ArrayList<Student> list, Scanner in, String name)
         editStudentAttributes(in, student);
     }
 
+    /***
```
Comment (tayponghee) [inline src/main/java/classify/commands/EditStudent.java:71]: Same here

File: src/main/java/classify/data/DataReader.java [hunk 3]
```diff
@@ -44,9 +44,20 @@ public static void restoreStudentList(String studentFileInput, ArrayList <Studen
         }
     }
 
+    //@@author blackmirag3
+
+    /**
+     * Initialises student array list from specified data file.
+     * Creates new data file if data file not found
+     *
```
Comment (tayponghee) [inline src/main/java/classify/data/DataReader.java:52]: Remove extra spacing here

File: src/main/java/classify/data/DataReader.java [hunk 4]
```diff
@@ -44,9 +44,20 @@ public static void restoreStudentList(String studentFileInput, ArrayList <Studen
...
+     *
+     * @param list ArrayList containing Students
+     * @param filePath String containing path to data file
+     * @throws IOException Thrown when there is error reading the data file.
+     */
```
Comment (tayponghee) [inline src/main/java/classify/data/DataReader.java:57]: Otherwise lgtm

**Review by tayponghee — CHANGES_REQUESTED** [review]

**Comment by Cryolian** [comment]
Please fix your merge conflicts

File: src/main/java/classify/commands/EditStudent.java [hunk 5]
```diff
@@ -30,17 +30,6 @@ public class EditStudent {
      * @param in         The scanner object to read user input.
      * @param name       The name of the student if the user has entered it when invoking the function.
      */
-    // @@author blackmirag3
```
Comment (tayponghee) [inline src/main/java/classify/commands/EditStudent.java:33]: Thanks for fixing!

**Review by tayponghee — APPROVED** [review]

**Pull request merged**